regolith/schemas.py

```diff
@@ -3736,7 +3793,11 @@
                                           f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
                            "required": False,
                            "type": "string",
-                           "eallowed": PROJECTUM_STATUS}
+                           "eallowed": PROJECTUM_STATUS},
+                "identifier": {"description": "label of kickoff",
```

Comment:
I am not sure I even remember what this is for.  Please can you try and figure it out by finding an example and  then make a better description ;)

Reply:
Not sure what is going on here.  My question is what is "identifier" and what is it for?  It may be here because someone has used it for something, but I don't recognize it and the description is not helpful.

Reply:
Yes, my mistake, I misunderstood. There is no example of identifier in the projecta database.... shall I delete this property?